util/autovector.h

```diff
@@ -323,13 +339,17 @@ class autovector {
 
  private:
   size_type num_stack_items_ = 0;  // current number of items
-  value_type values_[kSize];       // the first `kSize` items
+  alignas(alignof(
+      value_type)) char buf_[kSize *
```

Comment:
the formatting is quite weird, do you mind running "make format" before landing the PR?

Reply:
This was formatted by make format. Agree its a bit weird.